crates/viewer/re_time_panel/src/data_density_graph.rs

```diff
@@ -501,6 +605,29 @@ impl<'a> DensityDataAggregate<'a> {
         }
     }
 
+    fn add_chunk_point(&mut self, time: TimeInt, num_events: usize) {
+        let Some(x) = self.time_ranges_ui.x_from_time_f32(time.into()) else {
+            return;
+        };
+
+        self.density_graph.add_point(x, num_events as _);
+
+        if let Some(pointer_pos) = self.pointer_pos {
+            let is_hovered = {
+                // Are we close enough to the point?
+                let distance_sq = pos2(x, self.row_rect.center().y).distance_sq(pointer_pos);
+
+                distance_sq < self.interact_radius.powi(2)
+            };
+
+            if is_hovered {
+                if let Some(at_time) = self.time_ranges_ui.time_from_x_f32(pointer_pos.x) {
+                    self.hovered_time = Some(at_time.round());
+                }
```

Comment:
shouldn't this be

```suggestion
                self.hovered_time = Some(time);
```

?